pyproject.toml

```diff
@@ -417,7 +418,7 @@ runtime-evaluated-base-classes = ["pydantic.BaseModel", "anta.models.AntaTest.In
 ]
 "anta/cli/*" = [
   "PLR0913",  # CLI has many arguments defined in functions
-  "ANN401",   # TODO: Check if we can update the Any type hints in the CLI
+  "FBT001",   # CLI can have boolean as argument
```

Comment:
hmm can we not make the argument a keyword-only argument instead?

https://docs.astral.sh/ruff/rules/boolean-type-hint-positional-argument/